File: src/main/java/chapter1/FirstExample.java
package chapter1;

import io.reactivex.Observable;


public class FirstExample {

    public void emit() {
        Observable.just(1, 2, 3, 4, 5, 6)
                .subscribe(System.out::println);
    }

    public static void main(String args[]) {
        FirstExample demo = new FirstExample();
        demo.emit();
    }

}

File: src/main/kotlin/chapter1/FirstExample_.kt
package chapter1

import io.reactivex.Observable


class FirstExample_ {

    fun emit() {
        Observable.just(1, 2, 3, 4, 5, 6)
                .subscribe { println(it) }
    }

}


fun main(args: Array<String>) {
    val demo = FirstExample_()
    demo.emit()
}
